lib/broccoli/ember-app-with-packages.js

```diff
@@ -15,11 +15,7 @@ function EmberAppWithPackages(defaults, options) {
       // TODO: re-enable jshint once it's actually working fine, for now it just slows down the build
       // and running `ember test` it only jshints boot, but not the packages. We'll rely an npm script
       // for now
-      hinting: false,
-      fingerprint: {
-        // Disabling here since we do it at the end for *all* the assets
-        enabled: false,
```

Comment:
Yes. Assets were fingerprinted even if fingerprint option is disabled.  
Let me take that in a different PR and keep this focused on support for 2.10.